### ISSUE TITLE
Support for Android 15?

### ISSUE THREAD

**Issue by bschtl** [whew-inc/FakeGApps#22]
Is support for Android 15 planned?

**Comment by whew-inc** [comment]
Normally, yes. But since I don't actively use this project anymore on my personal device, I have to rely on other users to confirm compatibility with new Android versions.

Have you tried it on A15?

**Comment by bschtl** [comment]
Not yet, I wanted to ask in advance, as the compatibility of the module is explicitly stated. So it seems it's up to me to find out. Will need some time for this as I only have my personal device to give it a try.

Thank you for the continued support and development, that's very kind!

**Comment by bschtl** [comment]
I'm sorry this took so long, but I had several problems and the last one I just couldn't solve. I couldn't find any working method to debloat Play Services and Play Store on my device (CPH2581 OOS 15.0.0.406) as requisite for installing microG and therefore I am unable to confirm the compatibility of FakeGApps with Android 15.

Perhaps somebody else was more lucky and can confirm this?

**Comment by mherrmann3** [comment]
@bschtl: have your tried debloating systemless-ly with Magisk & [_De-Bloater_](https://sunilpaulmathew.github.io/De-Bloater/)? It works really well (at least up to Android 14) as preparation for installing _microG_ as user app. [This guide](https://xdaforums.com/t/guide-universal-systemless-microg-installation-on-stock-android-11-14.4686426/) details full procedure to replace _Google Play Services_ with _microG_ systemless-ly.

**Comment by bschtl** [comment]
@mherrmann3: Yes, on OOS14 I debloated systemless-ly, but with KernelSU and [KernelSUDebloater](https://gitlab.com/adrian.m.miller/kernelsudebloater). But after debloating I installed microG as system app via [MicroG Installer Revived](https://github.com/nift4/microg_installer_revived), which worked fine. 
After upgrading to OOS15 KernelSUDebloater didn't work anymore, so I gave De-Bloater a try. But debloating wit this led to being stuck at boot, which I could only solve by factory resetting. I am tempted to give it another try with the linked guide (which contains some new info for me) but really don't want to have to reset again, so I am not sure.
